serving/docs/lmi_new/user_guides/trt_llm_user_guide.md

```diff
@@ -97,39 +97,42 @@ print(outputs)
+
+Here are the advanced parameters that are available when using TensorRT-LLM. 
+Each advanced configuration is specified with a Configuration Type.
+`LMI` means the configuration is processed and used by LMI.
+`Pass Through` means the configuration is passed down to the library.
+If you encounter an issue with a `Pass Through` configuration, it is likely an issue with the underlying library and not LMI.
+
+| Item                                                          | LMI Version | Configuration Type | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          | Example value                                                                                                                                                               |
+|---------------------------------------------------------------|-------------|--------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| option.max_input_len	                                         | >= 0.25.0   | LMI	               | Maximum input token size you expect the model to have per request. This is a compilation parameter that set to the model for Just-in-Time compilation. If you set this value too low, the model will unable to consume the long input.                                                                                                                                                                                                                                                                                                                                                               | Default: `1024`                                                                                                                                                             |
```

Comment:
input and output lengths are also passed directly to library without modification. These are however used by LMI during inference to validate input parameters, so not sure how you want to classify them.

Reply:
I think I will update the description for `LMI` classification to include how LMI does some additional validation/translation before passing down information to the library.

In this case, since we're using it for validation above what trt provides, I think LMI is good